Source/OxyPlot/Rendering/RenderContext/IRenderContext.cs

```diff
@@ -186,6 +186,12 @@ public interface IRenderContext
         /// <param name="text">The text in the tool tip, or <c>null</c> if no tool tip should be shown.</param>
         void SetToolTip(string text);
 
+        /// <summary>
+        /// Updates render context state
+        /// </summary>
+        /// <remarks>This method is called at the beginning of each rendering.</remarks>
+        void BeginFrame();
```

Comment:
Despite of trying to avoid anything that could lead to inherent changes, I've added BeginFrame to IRenderContext. I felt almost free to do it because there had already been a method CleanUp which acts the same: it's invoked by the core, once per frame, has five empty implementations... Both of these methods could easily be removed from the interface and be implemented in CanvasRenderContext directly to be invoked from e.g. PlotView. But I'd like to rename CleanUp to EndFrame to make BeginFrame-EndFrame symmetric. As these methods are invoked by the core it would be nice to use them not only in WPF but in e.g. Xamarin or OxyPlot.SharpDX as well to manage platform specific resources. @objorke What do you think? Can anyone grant me permissions for such a breaking change? Or it's better to rollback the interface changes, leave everything as is and implement BeginFrame in WPF's CanvasRenderContext only?